### DISCUSSION TITLE
How to remove specific name table entry?

### DISCUSSION THREAD

**Discussion by ollimeier** [googlefonts/fontations#657] [Q&A] [answered]
Hi,

How do I remove a specific name table entry from a font file?
I would expect something similar to:
`name.name_record.insert(name_rec)`

something like:
`name.name_record.remove(3, 1, 1033, NameId::new(0))`

Thanks a lot in advance.

**Comment by cmyr** [comment, marked as answer]
This isn't really something that we've had to deal with yet, hence no implementation.

Your best bet might be to use the built in [`BTreeSet::retain`](http://doc.rust-lang.org/1.72.1/std/collections/struct.BTreeSet.html#method.retain) method on the `name_record` collection.

You could also construct the namerecord that you want to remove, and then pass it to `BTreeSet::remove`.

**Reply by ollimeier** [reply]
Thanks. 
Meanwhile I found a different solution. I created a new name table with `Name::default()` and added all entries but not the once I wanted to delete.
